api/src/api/opportunities_v1/opportunity_schemas.py

```diff
@@ -535,3 +535,39 @@ class OpportunitySearchResponseV1Schema(AbstractResponseSchema, PaginationMixinS
         OpportunityFacetV1Schema(),
         metadata={"description": "Counts of filter/facet values in the full response"},
     )
+
+
+class UserV1Schema(Schema):
+    user_id = fields.String(
+        metadata={
+            "description": "The internal ID of a user",
+            "example": "861a0148-cf2c-432b-b0b3-690016299ab1",
+        }
+    )
+    email = fields.String(
+        metadata={
+            "description": "The email address returned from Oauth2 provider",
+            "example": "js@gmail.com",
+        }
+    )
+    external_user_type = fields.String(
+        metadata={
+            "description": "The Oauth2 provider through which a user was authenticated",
+            "example": "login_gov",
+        }
+    )
+
+
+class OpportunityUserTokenResponseV1Schema(AbstractResponseSchema):
```

Comment:
The abstract response schema usage means we should have something like:

```
class UserTokenResponseSchema(AbstractResponseSchema):
     data = fields.Nested(SomeOtherSchema())
```

Because we want the response to look like:
```json
{
   "data": {
      // the actual meaningful stuff goes here
   },
   ...
}
```

Reply:
That makes sense. Fixed!